flake.nix

```diff
@@ -56,8 +54,13 @@
             mkShell (attrs // {
               shellHook = ''
                 export PATH=$PWD/scripts:$PWD/scripts/ci:$PATH
+              '' + pkgs.lib.optionalString pkgs.stdenv.isDarwin ''
+                export CC=gcc
```

Comment:
Should we not be more specific here? How do I know if `gcc` resolves to `x86_64-gcc` or `aarch64-gcc` in the cross case?

Reply:
the `export CC=gcc` only applied to darwin, and in darwin we don't have the cross compilation toolchains, so I think it's fine to do it here, although this is not ideal

Reply:
Ack